src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#index.html"
```

Comment:
The `href="#index.html"` is not a valid internal link. If you intend to link to a section within the page, use an ID that exists in the document. Otherwise, use a proper URL.